src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -11,9 +11,13 @@
     <EmbeddedResource Include="Features\Schema\Migrations\5.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\6.diff.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\6.sql">
-      <InputToMutableSqlGenerator>true</InputToMutableSqlGenerator>
       <InputToImmutableSqlGenerator>true</InputToImmutableSqlGenerator>
```

Comment:
>true [](start = 36, length = 4)

is it because we don't include ReferenceTokenCompositeSearchParamTableType_1 in 7.sql, but it's still used somehow in code?

Reply:
Yes, for backwards compat. First, we upgrade the code, which needs to continue to work against the V6 schema. Then, after we upgrade the schema, the code can start using the V7 constructs. 
So the code needs to have access to the older types.